front/app/containers/EventsPage/EventsViewer/index.tsx

```diff
@@ -2,25 +2,35 @@ import React, { memo, useState, useEffect } from 'react';
 
 // components
 import TopBar from './TopBar';
+import { Spinner } from 'cl2-component-library';
+import EventCard from 'components/EventCard';
 import Pagination from 'components/Pagination';
 
 // svg
 import noEventsIllustration from './NoEventsPicture.svg';
 
+// hooks
+import useEvents from 'hooks/useEvents';
+
 // styling
 import styled from 'styled-components';
 import { colors, fontSizes } from 'utils/styleUtils';
 
 // other
+import { IEventData } from 'services/events';
 import { sliceEventsToPage, getNumberOfPages } from './eventsViewerUtils';
+import { isNilOrError } from 'utils/helperUtils';
 
-const PlaceHolder = styled.div<{ first: boolean }>`
-  width: 100%;
-  height: 237px;
-  margin-top: ${({ first }) => (first ? '29px' : '39px')};
-  padding: 30px;
-  font-size: ${fontSizes.xxl}px;
-  border: 1px dotted;
+interface IStyledEventCard {
+  last: boolean;
+}
+
+const StyledSpinner = styled(Spinner)`
+  margin-top: 70px;
```

Comment:
I wouldn't center a spinner like this. That will not work well on e.g. a small mobile phone.

Reply:
I don't know off the top of my head how we do this in less-than-full-screen containers in other places so I would search the codebase for examples, but loading is something we should soon tackle systematically as suggested by Iva in the debt meeting.

Reply:
I will just remove the styling for now, it was not really my intention to center it, just to give it a bit of margin. But it looks fine without it too I guess.

Reply:
Well, spinners are kind of 2010, but they're all over the platform 😁